content/en/docs/concepts/security/overview.md

```diff
@@ -114,51 +112,49 @@ Quality of Service (and Cluster resource management) | https://kubernetes.io/doc
 Network Policies | https://kubernetes.io/docs/concepts/services-networking/network-policies/
 TLS For Kubernetes Ingress | https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
 
-
-
 ## Container
 
-In order to run software in Kubernetes, it must be in a container. Because of this,
-there are certain security considerations that must be taken into account in order
-to benefit from the workload security primitives of Kubernetes. Container security
-is also outside the scope of this guide, but here is a table of general
-recommendations and links for further exploration of this topic.
+Container security is outside the scope of this guide. Here are general recommendations and
+links to explore this topic:
 
 Area of Concern for Containers | Recommendation |
------------------------------- | ------------ |
-Container Vulnerability Scanning and OS Dependency Security | As part of an image build step or on a regular basis you should scan your containers for known vulnerabilities with a tool such as [CoreOS's Clair](https://github.com/coreos/clair/)
-Image Signing and Enforcement | Two other CNCF Projects (TUF and Notary) are useful tools for signing container images and maintaining a system of trust for the content of your containers. If you use Docker, it is built in to the Docker Engine as [Docker Content Trust](https://docs.docker.com/engine/security/trust/content_trust/). On the enforcement piece, [IBM's Portieris](https://github.com/IBM/portieris) project is a tool that runs as a Kubernetes Dynamic Admission Controller to ensure that images are properly signed via Notary before being admitted to the Cluster.
+------------------------------ | -------------- |
+Container Vulnerability Scanning and OS Dependency Security | As part of an image build, periodically scan your containers for known vulnerabilities.
```

Comment:
You can scan at build time and you can periodically scan built containers. I don't think you can periodically do a scan as part of the build process.

Reply:
Thanks for the review!